### PR TITLE
MINOR: change Streams integration test log levels

### PR DESCRIPTION
**Pull request by ableegoldman**
And reduce the less relevant logs

### CODE REVIEW

**Comment by mjsax** [comment]
Retest this please.

**Comment by ableegoldman** [comment]
It worked!! Results for the failed tests actually include the Streams logs 😭 

[Unit test](https://builds.apache.org/job/kafka-pr-jdk11-scala2.13/7405/testReport/junit/org.apache.kafka.streams/TestUnitTest/testTest/)
[Integration test](https://builds.apache.org/job/kafka-pr-jdk11-scala2.13/7405/testReport/junit/org.apache.kafka.streams.integration/TestIntegrationTest/testTest/)

File: streams/src/test/resources/log4j.properties [hunk 1]
```diff
@@ -18,4 +18,9 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
-log4j.logger.org.apache.kafka=INFO
+log4j.logger.kafka=WARN
```
Comment (guozhangwang) [inline streams/src/test/resources/log4j.properties]: Do we really want to degrade broker side logs to WARN?

Comment (ableegoldman) [reply streams/src/test/resources/log4j.properties]: We've been discussing this...the way I see it we have three options:
1) the current proposal (demote brokers to WARN)
2) just demote ZK to ERROR and see if that helps enough
3) do a finer grained control of the broker logs, ie demote some less helpful packages to WARN but leave others at INFO

I personally am fine with either 1 or 2 for now, as we can always go back and change it again if it's too little or too much. But I'd rather get fewer broker logs and some Streams logs than no Streams logs at all 🙂

**Comment by mjsax** [comment]
Retest this please.

**Comment by ableegoldman** [comment]
@mjsax didn't work :/  
Can you try again?

File: streams/src/test/resources/log4j.properties (hunk 1 above)
Comment (guozhangwang) [reply streams/src/test/resources/log4j.properties]: Got it, that makes sense.

Are there are particular Streams sub-packages we are interested in, or just `o.a.k.streams.integration` itself is good enough?

Comment (ableegoldman) [reply streams/src/test/resources/log4j.properties]: Hm yeah I guess we should try setting everything to DEBUG. Just trying to find the sweet spot where we can actually get the logs that are useful

**Comment by mjsax** [comment]
Retest this please.

**Comment by mjsax** [comment]
Retest this please.

**Comment by mjsax** [comment]
Retest this please.

**Comment by ableegoldman** [comment]
Well it looks like we still get too many logs from the brokers even at INFO level. I think we should just demote all broker logs to WARN and just bump it back up to INFO if really necessary

On the other hand, seems like turning DEBUG logs on for Streams caused over 100 tests to fail...not sure what that's about

**Comment by guozhangwang** [comment]
@ableegoldman probably because the too much logs cause some of the operations cannot be completed within the timeout; let's demote brokers to WARN and I will trigger jenkins again.

**Comment by ableegoldman** [comment]
Ok I totally forgot about this PR.

@guozhangwang @mjsax Let's just merge the demotion of broker/zookeeper logs and leave streams logs at INFO for now. If that's not sufficient for useful debugging (eg the `EosBetaUpgradeIntegrationTest`) then I'll take another look at fixing the tests which fail when DEBUG logging is turned on.

I did take a brief look at the failing tests, it's not a timeout or excessive logs or anything that seems at all correlated to the log level. It fails on an assertion of the value of `MockProcessorSupplier#capturedProcessors` -- seems pretty bizarre to me 🤷‍♀️

**Comment by mjsax** [comment]
Retest this please.

**Comment by guozhangwang** [comment]
SG. +1

**Comment by ableegoldman** [comment]
1 unrelated failure: `SaslPlaintextConsumerTest.testCoordinatorFailover`

**Pull request merged**